cmd/snap-exec/main.go

```diff
@@ -146,9 +151,16 @@ func snapExecApp(snapApp, revision, command string, args []string) error {
 
 	// run the command
 	fullCmd := filepath.Join(app.Snap.MountDir(), cmd)
-	if command == "shell" {
+	switch command {
+	case "shell":
 		fullCmd = "/bin/bash"
 		cmdArgs = nil
+	case "complete":
+		fullCmd = "/bin/bash"
+		cmdArgs = []string{
+			dirs.CompletionHelper,
+			filepath.Join(app.Snap.MountDir(), app.Completer),
```

Comment:
This is incorrect. The mount dir is different in the outside and on the inside of the snap execution environment. On the inside it is a /snap/... but on the outside it is variable.